sklearn/model_selection/_split.py

```diff
@@ -476,28 +476,23 @@ class GroupKFold(_BaseKFold):
     --------
     >>> import numpy as np
     >>> from sklearn.model_selection import GroupKFold
-    >>> X = np.array([[1, 2], [3, 4], [5, 6], [7, 8]])
-    >>> y = np.array([1, 2, 3, 4])
-    >>> groups = np.array([0, 0, 2, 2])
+    >>> X = np.array([[1, 2], [3, 4], [5, 6], [7, 8], [9, 10], [11, 12]])
+    >>> y = np.array([1, 2, 3, 4, 5, 6])
+    >>> groups = np.array([0, 0, 2, 2, 3, 3])
     >>> group_kfold = GroupKFold(n_splits=2)
     >>> group_kfold.get_n_splits(X, y, groups)
     2
     >>> print(group_kfold)
     GroupKFold(n_splits=2)
     >>> for train_index, test_index in group_kfold.split(X, y, groups):
-    ...     print("TRAIN:", train_index, "TEST:", test_index)
-    ...     X_train, X_test = X[train_index], X[test_index]
-    ...     y_train, y_test = y[train_index], y[test_index]
-    ...     print(X_train, X_test, y_train, y_test)
+    ...     train_group, test_group = groups[train_index], groups[test_index]
+    ...     print("TRAIN:\tIndex:", train_index, ", Group:", train_group)
```

Comment:
To remove the space between the comma and the `train_index` array:

```suggestion
    ...     print(f"TRAIN:\tIndex: {train_index}, Group: {train_group}")
```

This results in:

```
TRAIN:  Index: [2 3], Group: [2 2]
```